connect/src/web.ts

```diff
@@ -861,8 +891,19 @@ export class Ad4mConnectElement extends LitElement {
 
   render() {
     console.log(this.authState,  this.connectionState, this.uiState, this._isOpen);
-    if (this._isOpen === false) return null;
-    if (this.authState === "authenticated") return null;
+    if (this._isOpen === false) {
+      if (this.authState === "authenticated" && this._isMobile) {

```

Comment:
Also here, I think we could always show this new logo button. Or: we make it an option. But I think it could help us when dogfooding and debugging also in the web/desktop case..